Engine/Module/ModuleFileSystem.cpp

```diff
@@ -35,235 +71,170 @@ bool ModuleFileSystem::CleanUp()
 	return PHYSFS_deinit();
 }
 
-ModuleFileSystem::~ModuleFileSystem()
+void ModuleFileSystem::AddPath(Path* path)
 {
-	CleanUp();
+	assert(Exists(path->GetFullPath()));
+	assert(paths.find(path->GetFullPath()) == paths.end());
+	paths[path->GetFullPath()] = path;
 }
 
-char* ModuleFileSystem::Load(const char* file_path, size_t & size) const
-{ 
-	assert(Exists(file_path));
```

Comment:
Actually I think it would be a good idea to keep saving and loading with SDL, because if we want the game to be saved by the user we will need to do that in the user system probably. Ans Physfs is not going to help with that I think. Of course I'm not sure of this maybe is something to ask to the teachers.

Reply:
PhysFS sets a writable directory in which you can write all users data. Atm this path is set using `SDL_GetPrefPath("OnionGalaxy", TITLE)`, where TITLE="Little Orion Engine"